inc/search.class.php

```diff
@@ -5079,8 +5260,10 @@ static function addLeftJoin($itemtype, $ref_table, array &$already_link_tables,
     * @param array  $already_link_tables2  Array of tables already joined
     *
     * @return string Meta Left join string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addMetaLeftJoin($from_type, $to_type, array &$already_link_tables2,
+   public function addMetaLeftJoin($from_type, $to_type, array &$already_link_tables2,
```

Comment:
This method is used in some plugins (`fusioninventory` for instance), change should be listed in changelog.